common/colour/reports/pre-a11y-2.json

```diff
@@ -0,0 +1,444 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/conformance-report-en.json",
+
+  "@type": [ "earl:Assertion", "acr:AssessmentReport" ],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/GCWeb/common/colour/colour-en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Evaluation of how various text appears on different backgrounds.",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": ["#text-white", "#text-sm-left", "#text-sm-right", "#bg-primary", "#bg-success","#bg-info", "#bg-danger", "#bg-warning", "#text-muted", "#text-primary", "#text-success", "#text-info", "#text-danger", "#text-left", "#text-right", "#text-center", "#text-justify", "#text-nowrap", "#bg-cover", "#bg-center", "#bg-norepeat", "#bg-gctheme", "#bg-darker", "#bg-dark"]
```

Comment:
This is not a valid CSS selector. The value of a pointer can only be a string and not not an array of string.

So I tried this but it did returned nothing for the tested page.
```js
document.querySelectorAll( "#text-white, #text-sm-left, #text-sm-right, #bg-primary, #bg-success, #bg-info, #bg-danger, #bg-warning, #text-muted, #text-primary, #text-success, #text-info, #text-danger, #text-left, #text-right, #text-center, #text-justify, #text-nowrap, #bg-cover, #bg-center, #bg-norepeat, #bg-gctheme, #bg-darker, #bg-dark") 
```

Note: The "#" character select ID of an element and not the CSS class. The "." character do select the CSS class.

Reply:
Fixed in commit "Made requested changes"